fe1-web/res/adhoc-implementations/WalletCryptoLibrary.native.ts

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
I'm a bit surprised of the location.
`res` stands for `resources`, i.e. images, data, assets, etc.
Not a typical location for source code, and `adhoc-implementations` seems an odd name for a folder: technically all implementations are ad hoc :)
You might want to put this simply in a folder like `fe1-web/utils/myfile.ts`